ui-test-args.yaml

```diff
@@ -5,4 +5,104 @@ ui-tests:
   test: app/build/outputs/apk/androidTest/internal/debug/app-internal-debug-androidTest.apk
   device:
     - {model: lucye, version: 24}
-    - {model: blueline, version: 29}
\ No newline at end of file
+    - {model: blueline, version: 29}
+
+switch-ui-tests:
+  timeout: 30m
+  type: instrumentation
+  app: audioswitch/ftl/app-debug.apk
+  test: audioswitch/build/outputs/apk/androidTest/debug/audioswitch-debug-androidTest.apk
+  device:
+    - {model: A0001, version: 22}
```

Comment:
This is a lot of devices 😅 

Can you just run on one device from API 24-29? If you can, select a Samsung device from each level. You can select a Google device if a Samsung device isn't available on an API level.